web/wp-content/themes/dctx-v2/inc/jetpack.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-6oavcg.php, but found phpcs-scan-6OAvCg.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).